op-deployer/pkg/deployer/pipeline/init.go

```diff
@@ -136,23 +134,3 @@ func InitGenesisStrategy(env *Env, intent *state.Intent, st *state.State) error
 func immutableErr(field string, was, is any) error {
 	return fmt.Errorf("%s is immutable: was %v, is %v", field, was, is)
 }
-
-func displayWarning() error {
-	warning := strings.TrimPrefix(`
-####################### WARNING! WARNING WARNING! #######################
-
-You are deploying a tagged release to a chain with no pre-deployed OPCM.
-Due to a quirk of our contract version system, this can lead to deploying
-contracts containing unaudited or untested code. As a result, this 
-functionality is currently disabled.
-
-We will fix this in an upcoming release.
-
-This process will now exit.
```

Comment:
Trying to understand the context behind this warning and why are we removing it. Mind helping out?

Reply:
see the docs for an explanation: https://devdocs.optimism.io/op-deployer/user-guide/known-limitations.html

Reply:
the opcm re-releases all contracts at once which obviates the need for this warning.